src/modules/Update.rb

```diff
@@ -855,11 +859,11 @@ def restore_backup?
     # @return [String] a string holding the ID and VERSION_ID or empty if something went wrong
     def version_from(file)
       content = file.read
-      id = content[/^ID=.*/].split("=", 2).last
-      version_id = content[/^VERSION_ID=.*/].split("=", 2).last
+      id = content[/^ID=.*/].split("=", 2).last.gsub('"', '')
```

Comment:
NP: in this simple case you can use [String#tr](https://ruby-doc.org/core-2.5.1/String.html#method-i-tr) which is a bit faster than more complex `gsub`.